.github/actions/package_csharp/scripts/mac_build_csharp.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+set -ex
+
+export ITK_GLOBAL_DEFAULT_NUMBER_OF_THREADS=2
+
+echo "COREBINARYDIRECTORY: ${COREBINARYDIRECTORY}"
+echo "CTEST_SOURCE_DIRECTORY: ${CTEST_SOURCE_DIRECTORY}"
+
+read -r -d '' CTEST_CACHE << EOM || true
+CMAKE_PREFIX_PATH:PATH=${COREBINARYDIRECTORY}
+CMAKE_CXX_VISIBILITY_PRESET:STRING=hidden
+CMAKE_VISIBILITY_INLINES_HIDDEN:BOOL=ON
+CMAKE_OSX_DEPLOYMENT_TARGET=10.9
+SWIG_EXECUTABLE:FILEPATH=${COREBINARYDIRECTORY}/Swig/bin/swig
+BUILD_EXAMPLES:BOOL=ON
+BUILD_TESTING:BOOL=ON
+SimpleITK_BUILD_DISTRIBUTE:BOOL=ON
+SimpleITK_BUILD_STRIP:BOOL=1
+EOM
+
+export CTEST_CACHE
+export CTEST_BINARY_DIRECTORY="${GITHUB_WORKSPACE}/CSharp"
+
+javac -version
```

Comment:
Checking the java compiler version is likely a remnant of the java build script which isn't relevant for the C# script here. If observation is correct, please remove.

Reply:
done